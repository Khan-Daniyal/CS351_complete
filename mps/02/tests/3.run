echo abracadabra | ./mytr abc 123
