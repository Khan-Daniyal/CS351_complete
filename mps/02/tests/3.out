12r131d12r1
